coderd/proxyhealth/proxyhealth.go

```diff
@@ -0,0 +1,8 @@
+package proxyhealth
```

Comment:
Felt kinda weird to make a whole file for a single struct but idk where else to put it.